www/app/HardwareController.js

```diff
@@ -5276,7 +5312,7 @@ define(['app'], function (app) {
                 $("#hardwarecontent #divhttppoller").hide();
                 if (text.indexOf("Open Weather Map") >= 0)
                 {
-                    $("#hardwarecontent #hardwareparamsunderground #location").val("lat=53.40&lon=14.58");
+                    $("#hardwarecontent #hardwareparamsunderground #location").val(data["Password"]);
```

Comment:
This is occasion to test for https://www.domoticz.com/forum/viewtopic.php?p=116842#p116842